<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStock</class>
 <widget class="QWidget" name="FormStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table_stock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>981</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Malzeme Ara</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>251</width>
      <height>153</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_productCode">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Malzeme Kodu :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_productName">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Malzeme Adı :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txt_productCode"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txt_productName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_productMainGroup">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Malzeme Üst Grup :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_productSecondGroup">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Malzeme Alt Grup :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txt_productMainGroup"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txt_productSecondGroup"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="btn_list">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Listele</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_Filter1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Filtre 1</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sütun :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_columnFilter1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Değer :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_rowFilter1">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_Filter2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Filtre 2</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sütun :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_columnFilter2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Değer :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_rowFilter2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_displaySettings">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Görüntüleme</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_displaySettings"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_stokManageGuide">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Kılavuz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_filters">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>630</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filtre : Seçilmedi.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refreshFilters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sıfırla</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_totalRecord">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>630</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Toplam Kayıt : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_totalAmount">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>630</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Toplam Miktar : </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
